extensions/2.0/Khronos/KHR_lights_punctual/schema/scene. KHR_lights_punctual.schema.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This file should probably be removed since lights could be instantiated only on nodes.